.github/workflows/maven_test.yml

```diff
@@ -176,7 +176,7 @@ jobs:
           python-version: '3.11'
           architecture: x64
       - name: Install Python packages (Python 3.11)
-        if: (contains(matrix.modules, 'sql#core')) || contains(matrix.modules, 'connect')
+        if: contains(matrix.modules, 'resource-managers#yarn') || (contains(matrix.modules, 'sql#core')) || contains(matrix.modules, 'connect')
```

Comment:
cc @HyukjinKwon FYI

Reply:
cc @dongjoon-hyun , this can restore the Maven daily tests, but to enhance developer friendliness, some refactoring may be necessary for the newly added cases in `YarnClusterSuite`: https://github.com/apache/spark/pull/49107/files#r1945123620